packages/components/src/components/wizard/wizard.jsx

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import Step from './wizard-step.jsx';
+import Icon from '../icon';
+import FormProgress from '../form-progress';
+import DesktopWrapper from '../desktop-wrapper';
+import MobileWrapper from '../mobile-wrapper';
+
+const Wizard = React.forwardRef(
+    ({ steps, lbl_previous, has_prev, lbl_next, has_next, has_form_progress, className }, ref) => {
+        const [step, setStep] = React.useState(0);
+        const [lastStep] = React.useState(steps.length - 1);
+        const [component, setComponent] = React.useState(steps[0].component);
+
+        const nextStep = () => {
+            if (step + 1 > lastStep) return;
```

Comment:
(suggestion) Is it possible to refactor this condition to a variable like `is_last_step = step + 1 > lastStep`?